payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -51,6 +51,7 @@ internal class ElementsSessionJsonParser(
         )
 
         val merchantCountry = json.optString(FIELD_MERCHANT_COUNTRY)
+        val isEligibleForCardBrandChoice = parseCardBrandChoiceEligibility(json)
```

Comment:
Should we add a test for when this fails?

Reply:
Added as part of the latest commit, which also adds the missing `client-side PIs only` requirement.